webapp/maps_heatmaps/src/index.tsx

```diff
@@ -171,7 +171,11 @@ async function getNextDocs(index: number, first: boolean) {
 /*@param first Determines whether this is a new collection and the next docs should be from the beginning,
```

Comment:
Extend this comment to explain what the function does

Reply:
Changed.